docs/enterprise/service-accounts.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Service Accounts
+sidebarDepth: 1
+description: Protect internal machine resources with service accounts.
+---
+
+# Service Accounts
+
+Service accounts offer a protected and standardized method of authenticating machine-to-machine communication between services protected by Pomerium.
+
+## Create a Service Account
+
+::: tip
+Before you begin, confirm you are in the correct Namespace. A service account can only be used in the Namespace it was created in, including its children Namespaces.
+:::
+
+1. From the main menu, select **Service Accounts** under **CONFIGURE**. Click the **+ ADD SERVICE ACCOUNT** button:
+
+   ![An empty Service Accounts page](./img/add-service-account.png)
+
+1. Service accounts can be unique and exist only for Pomerium, or impersonate directory users from your IdP.
+
+   ::::: tabs
+   :::: tab Unique
+   Give the user a unique ID. Consider referencing the Namespace you're creating it under, for easier reference later. Optionally set an expiration date:
```

Comment:
We should note that the User ID value here would correspond to the `User` criteria on the policy screen.

E.G. If they want to say 

```
allow:
  or:
    - user:
        is: "whatever unique user id from above"
```

Reply:
(Hopefully) addressed in b04b67f146194f8ae58ed97f29ba88b50bda9c73.